components/library/libspf2/libspf2.p5m

```diff
@@ -45,6 +46,7 @@ file path=usr/include/spf2/spf_record.h
 file path=usr/include/spf2/spf_request.h
 file path=usr/include/spf2/spf_response.h
 file path=usr/include/spf2/spf_server.h
+file path=usr/lib/$(MACH64)/libspf2.a
 link path=usr/lib/$(MACH64)/libspf2.so target=libspf2.so.2.1.0
```

Comment:
We don't want to deliver static libraries. We only deliver them when no alternative is available.

Reply:
@nipos please set `DROP_STATIC_LIBRARIES` to `yes` in the `Makefile` (before the `common.mk` include) so the static library is removed from `sample-manifest.p5m` too.  Thanks.